LICENSE.rst

```diff
@@ -6,23 +6,20 @@ NengoBones license
 
 Copyright (c) 2018-2021 Applied Brain Research
 
-NengoBones is made available under a proprietary license
-that permits using, copying, sharing, and making derivative works from
-NengoBones and its source code for any non-commercial purpose,
-as long as the above copyright notice and this permission notice
-are included in all copies or substantial portions of the software.
-
-If you would like to use NengoBones commercially,
-licenses can be purchased from Applied Brain Research.
-Please contact info@appliedbrainresearch.com for more information.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
+NengoBones is made available under a proprietary license, the
+"ABR TECHNOLOGY LICENSE AND USE AGREEMENT" (the "ABR License"). The
+ABR License is available for download at
+`<https://www.appliedbrainresearch.com/license>`_. The ABR License is
```

Comment:
Can we include a brief summary here (something to convey that it's free for non-commercial use)? That license file is a lot to dig through, if I saw that I would basically just say "I'm not confident parsing this long legal document, I'll use a different package".